core/src/main/java/com/linecorp/armeria/internal/common/util/ChannelUtil.java

```diff
@@ -63,9 +80,9 @@
         try {
             // Use reflection, just in case a user excluded netty-transport-native-epoll from the dependencies.
             builder.add((ChannelOption<?>) Class.forName(
-                    "io.netty.channel.epoll.EpollChannelOption", false,
```

Comment:
My understanding is that this sort of string is automatically shaded at least by standard tools. Are there shading tools that don't do this?

Reply:
For me it often means annoyance like "_io.foo.Bar".substring(1) to avoid shading which is annoying :)

Reply:
We use a very old version of JarJar which doesn't do anything with strings. :sweat_smile: